x/tokenfactory/keeper/keeper.go

```diff
@@ -63,11 +71,15 @@ func (k Keeper) GetCreatorsPrefixStore(ctx sdk.Context) sdk.KVStore {
 	return prefix.NewStore(store, types.GetCreatorsPrefix())
 }
 
+// Set the wasm keeper.
+func (k *Keeper) SetContractKeeper(contractKeeper types.ContractKeeper) {
+	k.contractKeeper = contractKeeper
+}
+
 // CreateModuleAccount creates a module account with minting and burning capabilities
 // This account isn't intended to store any coins,
 // it purely mints and burns them on behalf of the admin of respective denoms,
 // and sends to the relevant address.
 func (k Keeper) CreateModuleAccount(ctx sdk.Context) {
-	moduleAcc := authtypes.NewEmptyModuleAccount(types.ModuleName, authtypes.Minter, authtypes.Burner)
-	k.accountKeeper.SetModuleAccount(ctx, moduleAcc)
+	k.accountKeeper.GetModuleAccount(ctx, types.ModuleName)
 }
```

Comment:
wat? in a CreateSmth method we just call a getter and ignore the return value? something's definitely wrong here

Reply:
I think it creates module account under the hood: https://github.com/cosmos/cosmos-sdk/blob/8b60d4112b32a88a20fbe25e02d24a52aeeaa79e/x/auth/keeper/keeper.go#L214